server/src/main/resources/application.conf

```diff
@@ -36,6 +36,14 @@ persistence {
   snapshot-interval {
     voyage-manifest: ${?SNAPSHOT_INTERVAL_VM}
   }
+  snapshot-megabytes {
+    base-arrivals: ${?SNAPSHOT_MB_BASE_ARRIVALS}
```

Comment:
presumably this will need a puppet change too.

Reply:
They're actually optional as sensible defaults are set in the app. It was handy for testing and could be used if we find the values need tweaking on a port by port basis